### PR TITLE
[Feature] #42 #43 #44 Add DictionaryMainView UI

### LINKED ISSUE #42: [Feature] Grid UI construct reflecting the hi-fi prototype
# 유저는 장비군을 확인할 수 있다.
- 장비군을 확인할 수 있는 Grid UI를 구성한다.
   - Reflect the hi-fi design prototype

## 🤔 Completion conditions

- [ ] Grid UI components struct
- [ ] Add image
- [ ] Add font size, padding, etc.

## 👀 Related issues

- Related issue1 (#17)

### LINKED ISSUE #43: [Feature] Load the equipment data
# 유저는 장비군을 확인할 수 있다.
- Load the equipment group data

## 🤔 Completion conditions

- [ ] Connect the data

## 👀 Related issues

- Related issue1 (#44)

### LINKED ISSUE #44: [Feature] Construct grid UI according to the imported data
# 유저는 장비군을 확인할 수 있다.
- Construct the grid according to the imported data

## 🤔 Completion conditions

- [ ] Data connection

## 👀 Related issues

- Related issue1 (#43)

### PR DESCRIPTION
**Pull request by mijinc**
# Issue Number
🔒 Close #42
🔒 Close #43
🔒 Close #44

## Changes

- Changed files
Enum(
campair/Enum/Category.swift
)
campair/campair.xcodeproj/project.pbxproj
Views(
campair/campair/View/EquipmentDictionary/DictionaryMainView.swift
campair/campair/View/EquipmentDictionary/EquipmentDictionaryDetailView.swift
)
Asset(Image) (
campair/campair/Application/Assets.xcassets/CategoryBackground.imageset/CategoryBackground.svg
campair/campair/Application/Assets.xcassets/CategoryBackground.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_1.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_1.imageset/paint_1.svg
campair/campair/Application/Assets.xcassets/paint_10.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_10.imageset/paint_10.svg
campair/campair/Application/Assets.xcassets/paint_11.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_11.imageset/paint_11.svg
campair/campair/Application/Assets.xcassets/paint_12.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_12.imageset/paint_12.svg
campair/campair/Application/Assets.xcassets/paint_13.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_13.imageset/paint_13.svg
campair/campair/Application/Assets.xcassets/paint_14.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_14.imageset/paint_14.svg
campair/campair/Application/Assets.xcassets/paint_15.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_15.imageset/paint_15.svg
campair/campair/Application/Assets.xcassets/paint_2.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_2.imageset/paint_2.svg
campair/campair/Application/Assets.xcassets/paint_24.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_24.imageset/paint_24.svg
campair/campair/Application/Assets.xcassets/paint_3.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_3.imageset/paint_3.svg
campair/campair/Application/Assets.xcassets/paint_4.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_4.imageset/paint_4.svg
campair/campair/Application/Assets.xcassets/paint_5.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_5.imageset/paint_5.svg
campair/campair/Application/Assets.xcassets/paint_6.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_6.imageset/paint_6.svg
campair/campair/Application/Assets.xcassets/paint_7.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_7.imageset/paint_7.svg
campair/campair/Application/Assets.xcassets/paint_8.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_8.imageset/paint_8.svg
campair/campair/Application/Assets.xcassets/paint_9.imageset/Contents.json
campair/campair/Application/Assets.xcassets/paint_9.imageset/paint_9.svg
)

## New features

- Add CategoryView 
- Add SVG Images
- Create Enum folder

## Review points

- Views and structures are complex

## Questions

- How to structure the CLEAN structures?

## References

- https://github.com/DeveloperAcademy-POSTECH/TipPlace

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Review by rriver2 — COMMENTED** [review]
미진초이 ~~ Congratulation your first PR with 욕망의 볼따구 ~~ 🌷🎉❤️

**Comment by Gobans** [comment]
Congratulation your first PR!! 🥳🥳🥳 welcome to greedy code review!

File: campair/campair/View/EquipmentDictionary/DictionaryMainView.swift [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    let imageUrl: String
+    let categoryEnum: Category
+}
+
+struct CategoryButtonView: View {
+    let title: Category
+    let imageUrl: String
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            Button {
```
[the remark this answers is not in the document]
Comment (mijinc) [reply campair/campair/View/EquipmentDictionary/DictionaryMainView.swift:21]: Thank you! Done!

File: campair/campair/View/EquipmentDictionary/DictionaryMainView.swift [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            } label: {
+                NavigationLink(destination: SelectingEquipmentView()) {
+                    VStack {
+                        Image(imageUrl)
```
[the remark this answers is not in the document]
Comment (mijinc) [reply campair/campair/View/EquipmentDictionary/DictionaryMainView.swift:25]: Fixed!

File: campair/campair/View/EquipmentDictionary/DictionaryMainView.swift [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        ZStack {
+            VStack {
+                HStack {
+                    Text("여행의 시작은\n장비 준비부터")
+                        .font(.largeTitle)
+                        .fontWeight(.bold)
+                        .foregroundColor(Color.black)
+                        .multilineTextAlignment(.leading)
+                        .padding([.leading], 20.0)
+                        .padding([.top], 36.0)
```
[the remark this answers is not in the document]
Comment (mijinc) [reply campair/campair/View/EquipmentDictionary/DictionaryMainView.swift:81]: Oh, thank you! Fixed all!

File: campair/campair/View/EquipmentDictionary/EquipmentDictionaryDetailView.swift [hunk 4]
```diff
@@ -60,7 +60,7 @@ struct SelectingEquipmentDictionaryView: View {
 struct SelectingEquipmentView: View {
     var body: some View {
         Button {
-            // TODO: Add Handling
+            //ToDo: Add Handling
```
Comment (sustainable-git) [inline campair/campair/View/EquipmentDictionary/EquipmentDictionaryDetailView.swift:63]: 스크린샷 2022-06-17 오전 10 41 16
The reason we use `// TODO:` is that we can find comments by hierarchy

File: campair/campair/View/EquipmentDictionary/DictionaryMainView.swift [hunk 5]
```diff
@@ -0,0 +1,116 @@
+//
+//  DictionaryMainView.swift
+//  campair
+//
+//  Created by Mijin CHOI on 2022/06/15.
+//
+
+import SwiftUI
+
+struct CategoryButton: Identifiable {
```
Comment (sustainable-git) [inline campair/campair/View/EquipmentDictionary/DictionaryMainView.swift:10]: The name, CategoryButton, seems like a View
I think just Category is fine

**Review by sustainable-git — APPROVED** [review]
GGOODD!

File: campair/campair/View/EquipmentDictionary/EquipmentDictionaryDetailView.swift (hunk 4 above)
Comment (mijinc) [reply campair/campair/View/EquipmentDictionary/EquipmentDictionaryDetailView.swift:63]: Thank you! I changed it.

File: campair/campair/View/EquipmentDictionary/DictionaryMainView.swift (hunk 5 above)
Comment (mijinc) [reply campair/campair/View/EquipmentDictionary/DictionaryMainView.swift:10]: Thank you! but because of the enum setting, I use this name just for now.

**Pull request closed without merging**